src/components/picker.tsx

```diff
@@ -45,9 +45,9 @@ export default class Picker extends React.Component<Props, State> {
 
     this.state = {
       search: '',
+      sort: 'id',
```

Comment:
yeah.  This is related. I really want you to be able to use an enum here...  I'm wondering if there is a good way to do that.